tests/rptest/tests/topic_creation_test.py

```diff
@@ -210,6 +210,49 @@ def test_create_topic_with_single_configuration_property(self):
             cfgs = rpk.describe_topic_configs(topic=name)
             assert str(cfgs[p][0]) == str(property_value)
 
+    @cluster(num_nodes=3)
```

Comment:
This test could probably be single node to make it a bit cheaper to run

Reply:
Done.